IntroToModelling.Rmd

```diff
@@ -0,0 +1,1060 @@
+  add_predictions(model, var = "predicted_height") %>% # adds the fitted values
+  add_residuals(model) #  adds the residuals
+
+sample
+```
+
+We can then use the predicted outcomes to plot the best fit line:
+
+```{r, echo = TRUE, eval = TRUE}
+sample %>%
+  ggplot(aes(x = weight)) +
+  geom_point(aes(y = height)) +
+  geom_line(aes(y = predicted_height), col = "red") +
+  theme_minimal()
+```
+
+Note that we could also use `geom_smooth()` or `geom_abline()` to plot this line. However, the method above using the `add_predictions()` values is much more general and can be used when we move onto multiple regression.
+
+We can also use `add_predictions()` to get predicted outcome values for unseen data, or to generate outcome values for a sequence of data points. e.g. to get predicted `height` values for a `weight` in the range from 50 to 120kg, we could do:
+
+```{r, echo = TRUE, eval = TRUE}
+# create a one-column data frame of weight values
+weights_predict <- tibble(
+  weight = 50:120
+)
+# use our model to predict corresponding height values
+weights_predict %>%
+  add_predictions(model)
+```
+
+<br>
+
+### Interpreting the slope
+
+$$
+height = 102.1758 + 0.9336 × weight
+$$
+
+The gradient $b_1$ (or we could refer to it as $b_{weight}$ as it affects the $weight$ variable) is often called the **regression coefficient**. This term appears a lot in linear regression and it sounds scarier than it is.
+
+The regression coefficient here links the outcome and predictor variables in the following way:
+
+::: {style="text-align: center;"}
+**A 1kg increase in weight changes the predicted height by 0.9336cm**
+:::
+
+<br>
+
+## Model diagnostics: part 1
+
+So we've built a model that helps us explain our data and predict unseen values. But is it a good model? And what do we mean by a model being 'good'?
+
+In this section, we'll cover how to examine the output from our `lm()` model and see if it's a good fit for our data (and a good prediction tool!).
+
+<br>
+
+### Using `summary()`
+
+The base R function, `summary()`, provides an overview of the inner workings of the model:
+
+```{r, echo = TRUE, eval = TRUE}
+summary(model)
+```
+
+<ol>
+
+<li>The model formula, with outcome variable \~ predictor variable.</li>
+
+<li>The minimum, medium, maximum and quartile values of the residuals. We don't need to worry about these too much.</li>
+
+<li>
+
+The **coefficients** section:
+
+<ul>
+
+<li>The `Estimate` column shows us the intercept and the coefficient (gradient) for the `weight` variable, i.e. $b_0$ and $b_1$.</li>
+
+<li>The `Std. Error` and `t value` columns, which we don't need to worry about for our purposes.</li>
+
+<li>The `Pr(>|t|)` column shows us the p-values for the intercept and coefficient. If you're unfamiliar with this term, we will cover them shortly don't worry. For now: if p is very small, we consider this feature of the model to have a significant effect on the outcome variable.</li>
+
+</ul>
+
+</li>
+
+</ul>
+
```

Comment:
Maintain the use of markdown where possible. 

The description of the output could also link up with the section titles to make identification easier, e.g.:
```suggestion
1. **Call:** the model formula, with outcome variable \~ predictor variable.
2. **Residuals:** the minimum, median, maximum and quartile values of the residuals. We don't need to worry about these too much.
3. **Coefficients:**
  - The `Estimate` column shows us the intercept and the coefficient (gradient) for the `weight` variable, i.e. $b_0$ and $b_1$.
  - The `Std. Error` and `t value` columns, which we don't need to worry about for our purposes.
  - The `Pr(>|t|)` column shows us the p-values for the intercept and coefficient. If you're unfamiliar with this term, we will cover them shortly don't worry. For now: if p is very small, we consider this feature of the model to have a significant effect on the outcome variable.
4. **Signif. codes:** ...
5. **Residual standard error:** ...
6. **Multiple R-squared:** ...
7. **F-statistic:** ...
```